### PR TITLE
Fix editor shortcuts on mac replacing ctrl with cmd

### PR DESCRIPTION
**Pull request by gurgendav**
* Replace shortcut for mac is set to CMD + ALT + F because CMD + H is used for "hide window" system command

Fixes: #78

### CODE REVIEW

**Comment by danwalmsley** [comment]
I think we added an api to avalonia that essentially does GetPlatformKey, I'll post a link to it.

**Comment by gurgendav** [comment]
Okay. Please give me the link and I'll update my pull request

File: src/AvaloniaEdit/Editing/CaretNavigationCommandHandler.cs [hunk 1]
```diff
@@ -108,7 +108,7 @@ static CaretNavigationCommandHandler()
             AddBinding(EditingCommands.MoveToDocumentEnd, InputModifiers.Control, Key.End, OnMoveCaret(CaretMovementType.DocumentEnd));
             AddBinding(EditingCommands.SelectToDocumentEnd, InputModifiers.Control | InputModifiers.Shift, Key.End, OnMoveCaretExtendSelection(CaretMovementType.DocumentEnd));
 
-            AddBinding(ApplicationCommands.SelectAll, InputModifiers.Control, Key.A, OnSelectAll);
+            AddBinding(ApplicationCommands.SelectAll, InputModifiers.None, Key.None, OnSelectAll);
```
Comment (aelij) [inline src/AvaloniaEdit/Editing/CaretNavigationCommandHandler.cs]: Can you explain this change?

Comment (gurgendav) [reply src/AvaloniaEdit/Editing/CaretNavigationCommandHandler.cs]: Sorry for really late response. 
It looked like some kind of workaround that key combinations for all ApplicationCommands was declared in class but for "SelectAll" doesn't. So I moved declaration to ApplicationCommands

Comment (aelij) [reply src/AvaloniaEdit/Editing/CaretNavigationCommandHandler.cs]: Then I don't think this line is needed any more.

Comment (gurgendav) [reply src/AvaloniaEdit/Editing/CaretNavigationCommandHandler.cs]: It's needed to add command binding. 

I created overload of AddBinding method which doesn't need those extra arguments.

**Comment by aelij** [comment]
@gurgendav can you please merge with master? There are conflicts. Thanks

**Comment by garyhertel** [comment]
It looks like a lot of this fix got reverted in the merge to mainline

**Pull request merged**